templates/shield-study/package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "shield-studies-addon-template",
-  "description": "Template Shield Study",
-  "version": "2.0.0",
+  "name": "{{id}}",
+  "description": "@TODO",
+  "version": "1.0.0",
   "author": "Mozilla",
   "bugs": {
     "url": "https://github.com/mozilla/shield-studies-addon-template/issues"
   },
   "dependencies": {
-    "shield-studies-addon-utils": "github:mozilla/shield-studies-addon-utils#release/5.0.4"
+    "shield-studies-addon-utils": "github:mozilla/shield-studies-addon-utils"
```

Comment:
This possibly fixes #3; "pathspec 'release/5.0.4' did not match any file(s) known to git", but possibly not in the right way. This will just pull the **master** branch, which may be risky. We could just switch to semver and pull the latest version from npm instead of trying to wrestle w/ github branches/releases.